apps/cms/src/migrations/20240616_101649_Add_displayTitle_to_database.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  type MigrateUpArgs,
+  type MigrateDownArgs,
+} from "@payloadcms/db-mongodb";
+import { type PayloadRequest } from "payload/types";
+import { CommitteeMembers } from "../collections/committees/committee-members";
+import { NewsItems } from "../collections/weekly-newsletters/news-items";
+
+export async function up({ payload }: MigrateUpArgs): Promise<void> {
+  const req = {} as PayloadRequest;
+
+  const committeeMembers = await payload.find({
+    collection: CommitteeMembers.slug,
+    pagination: false,
+  });
+
+  const newsItems = await payload.find({
+    collection: NewsItems.slug,
+    pagination: false,
+  });
+
+  for (const member of committeeMembers.docs) {
+    await payload.update({
+      collection: CommitteeMembers.slug,
+      id: member.id,
+      data: {
+        displayTitle: `${member.name}, ${member.title}`,
+      },
+      req,
+    });
+  }
+
+  for (const item of newsItems.docs) {
+    const date = new Date(item.date ?? "");
+    const displayTitle = isNaN(date.getTime())
+      ? item.title
+      : `${item.title} - ${date.toLocaleDateString("fi-FI")}`;
+    await payload.update({
+      collection: NewsItems.slug,
+      id: item.id,
+      data: {
```

Comment:
Hmm, this change for newsItems probably breaks localization in the admin UI and shows the last saved language version :thinking: 

It could be fine, but we could also make the field localized and save both locales here, but I wonder if that has even more implications for the search...